.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Bug Report
+about: Identify a bug in the Agent codebase
+title: "[BUG] "
+labels: team/triage
+assignees: ''
+
+---
+
+<!--
+If you have identified a specific bug in the Agent codebase, please describe it here.  If the
+issue requires further debugging or investigation, that will be best accomplished via support.  Please please contact Datadog [support](http://docs.datadoghq.com/help/) and send them a [flare](https://docs.datadoghq.com/agent/troubleshooting/#send-a-flare) demonstrating the issue.
```

Comment:
Tiny nit: Single space after period should be used as the style (applies to all 3 templates)